remote/remote.go

```diff
@@ -956,3 +958,7 @@ func isJSON(str string) bool {
 	var js json.RawMessage
 	return json.Unmarshal([]byte(str), &js) == nil
 }
+
+func isJSONObject(str string) bool {
+	return -1 != strings.Index(str, "{") && isJSON(str)
```

Comment:
What do you think about using `strings.HasPrefix` to improve a bit the readability?

```suggestion
	return strings.HasPrefix(str, "{") && isJSON(str)
```

Do we care about JSON arrays here? If so, I think we should also check if the string starts with `[`.

Reply:
Good catch. 
I have a change yet to be pushed that use the contains. HasPrefix is way better though.
Considering the JSON spec we should take into account the whitespace too. 
I'll think about it because with HasPrefix we risk to miss some valid JSON along the way. 

For example this JSON with extra spaces is valid.
 <pre>    {"test":1}    </pre>